index.js

```diff
@@ -1,14 +1,15 @@
+import logger from './src/util/logger';
```

Comment:
Also we try to avoid the `util` pattern so it's better to put it plain in the src folder